soc/SoC_OCV.py

```diff
@@ -1,16 +1,20 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import itertools
-from sklearn import datasets, linear_model, metrics
-from sklearn.preprocessing import PolynomialFeatures
-from sklearn.pipeline import make_pipeline
+
```

Comment:
Would definitely encourage you to delete these lines instead of commenting them out (if they're no longer used)